backend/mempool-config.sample.json

```diff
@@ -66,13 +66,22 @@
   },
   "SOCKS5PROXY": {
     "ENABLED": false,
+    "USE_ONION": true,
     "HOST": "127.0.0.1",
     "PORT": 9050,
     "USERNAME": "",
     "PASSWORD": ""
   },
   "PRICE_DATA_SERVER": {
     "TOR_URL": "http://wizpriceje6q5tdrxkyiazsgu7irquiqjy2dptezqhrtu7l2qelqktid.onion/getAllMarketPrices",
-    "CLEARNET_URL": "https://price.bisq.wiz.biz/getAllMarketPrices"
+    "CLEARNET_URL": "https://price.bisq.wiz.biz/getAllMarketPrices",
+    "BISQ_URL": "https://bisq.markets/api",
+    "BISQ_ONION": "http://bisqmktse2cabavbr2xjq7xw3h6g5ottemo5rolfcwt6aly6tp5fdryd.onion/api"
+  },
+  "EXTERNAL_DATA_SERVER": {
+    "MEMPOOL_API": "https://mempool.space/api/v1",
+    "MEMPOOL_ONION": "http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion/api/v1",
+    "LIQUID_API": "https://liquid.network/api/v1",
+    "LIQUID_ONION": "http://liquidmom47f6s3m53ebfxn47p76a6tlnxib3wp6deux7wuzotdr6cyd.onion/api/v1"
   }
```

Comment:
Exposed static URLs used to fetch resources to `mempool-config.json` ( So it could be updated by users if they have an alternative & Would be better to manage external resources )